tests/globalhelper/init.go

```diff
@@ -2,35 +2,43 @@ package globalhelper
 
 import (
 	"fmt"
-	"os"
 
 	"github.com/golang/glog"
 	testclient "github.com/test-network-function/cnfcert-tests-verification/tests/utils/client"
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/config"
 )
 
 var (
-	APIClient     *testclient.ClientSet
-	Configuration *config.Config
+	apiclient *testclient.ClientSet
+	conf      *config.Config
 )
 
-func init() {
-	if os.Getenv("UNIT_TEST") != "" {
-		Configuration = &config.Config{}
-
-		return
+func GetAPIClient() *testclient.ClientSet {
+	if apiclient != nil {
+		return apiclient
 	}
 
-	var err error
-	APIClient, err = config.DefineClients()
-
+	tempClient, err := config.DefineClients()
```

Comment:
why having tempClient and Configuration variable? why not:
var err error
apiclient, err = config.DefineClients()

Reply:
Good call, was a leftover.  I'll update it.